### PR TITLE
Changes to push Telegram attestation to production

### PR DESCRIPTION
**Pull request by shahthepro**
- Should fix the white screen issue on Origin Wallet after successful attestation
- Support for different bots on staging and production

### CODE REVIEW

File: dapps/marketplace/webpack.config.js [hunk 1]
```diff
@@ -21,9 +21,16 @@ try {
...
-  ? 'originprotocol_verification_bot'
-  : 'origin_protocol_test_bot'
+const isStaging = process.env.NAMESPACE === 'staging'
+const isDev = process.env.NAMESPACE === 'dev'
+
+let TELEGRAM_BOT_USERNAME = 'origin_protocol_test_bot'
+
+if (isStaging) {
+  TELEGRAM_BOT_USERNAME = 'originprotocol_verification_bot'
+} else if (isProduction && !isDev) {
```
Comment (sparrowDom) [inline dapps/marketplace/webpack.config.js]: is `!isDev` required?

**Comment by sparrowDom** [comment]
Though It would be cool to create an issue, to make a proper solution of this hack. Too many hacks can create a shaky foundation

**Comment by shahthepro** [comment]
@sparrowDom Yeah, I'll raise an issue on this. BTW, I have replaced the "clickjack" hack we were doing with a more appropriate way in this PR: https://github.com/OriginProtocol/origin/pull/2966

File: dapps/marketplace/webpack.config.js (hunk 1 above)
Comment (shahthepro) [reply dapps/marketplace/webpack.config.js]: Not necessary. We haven't configured webhooks on dev till now though.

**Pull request merged**